src/Symfony/Component/Clock/SleepableClockInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Clock;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+interface SleepableClockInterface
```

Comment:
Nitpicking the interface and method name. `delay` and `DelayableClockInterface` would be semantically more correct. `sleep` is from a process pov, `delay` is about the clock.

Reply:
I thought about "pause" also, but I kept sleep because to me this is the most expected and discoverable.
Any other name would introduce confusion IMHO, eg "delay": ":thinking: does that mean async?"

Reply:
My understanding may be biased. The method actually suspend the execution of the process; it will be understood as a wrapper for `sleep`. Ok for keeping this wording.